site/content/en/docs/python-api/python-api-examples/tiling.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Tiling'
+linkTitle: 'tiling'
+description: ''
+---
+
+Description
+-----------
+
+This API allows you to transform a dataset by tilling. This transform is known to be useful for detecting small objects in high-resolution input images <a href="#ref1">[1]</a>. In general, we resize the input image to be smaller the neural network can afford, in terms of memory and computational cost. This practice makes small objects so small that the neural network cannot distinguish them. The tiling transform increases the size of a small object's area relative to the network's receptive area to make it distinguishable. Therefore, this transform allows you to easily construct a dataset to train a model to detect small objects in high resolution images.
```

Comment:
- This API allows you to transform a dataset into a tiled one. In general, we resize the input image to be small enough for the network to afford in terms of memory and computational cost.
- high resolution images -> high-resolution images

Reply:
Fixed.